packages/trader/src/index.js

```diff
@@ -1,12 +1,6 @@
-import 'babel-polyfill';
+// import 'babel-polyfill';
 import 'promise-polyfill';
-import '_common/lib/polyfills/nodelist.foreach';
-import '_common/lib/polyfills/element.closest';
```

Comment:
safe to remove these polyfills?

Reply:
@kelly-binary Yes, there's more polyfill that we use that can be safely removed. These are the ones that were removed from Core as well.